app/src/main/java/uk/co/ribot/androidboilerplate/util/EventPosterHelper.java

```diff
@@ -3,24 +3,22 @@
 import android.os.Handler;
 import android.os.Looper;
 
-import com.squareup.otto.Bus;
-
 import javax.inject.Inject;
 
 /**
  * Provides helper methods to post event to an Otto event bus
  */
 public class EventPosterHelper {
```

Comment:
This class is not longer needed with the `RxEventBus` because you can handle threads etc as any other Observable using schedulers.